src/sandbox-client.ts

```diff
@@ -573,24 +588,20 @@ export class SandboxClient {
               .baseUrl?.replace("api", "global-scheduler");
 
             await fetch(
-              `${baseUrl}/api/v1/cluster/${data.id}?preferredManager=${preferredManager}`
+              `${baseUrl}/api/v1/cluster/${session.id}?preferredManager=${preferredManager}`
             ).then((res) => res.json());
           }
 
           return {
-            bootupType: data.bootup_type as
-              | "RUNNING"
-              | "CLEAN"
-              | "RESUME"
-              | "FORK",
-            pitcherURL: data.pitcher_url,
-            workspacePath: data.workspace_path,
-            userWorkspacePath: data.user_workspace_path,
-            pitcherManagerVersion: data.pitcher_manager_version,
-            pitcherVersion: data.pitcher_version,
-            latestPitcherVersion: data.latest_pitcher_version,
-            pitcherToken: data.pitcher_token,
-            cluster: data.cluster,
+            bootupType: "RESUME",
```

Comment:
Is this always a `RESUME`? I think we might want to remove this field for the user though, so should be fine.

Reply:
Yeah, this is not technically correct, but it did not look like relevant information for the user and it just cleaned up this passing of `SessionData`. We could maybe add `bootupType` on the `SessionData`, but it is a bit weird to persist a bootupType 😅 

Can talk more about it 😄